src/templates/user/telegram_bot.html.tt

```diff
@@ -0,0 +1,12 @@
+[% PROCESS components/forms.html.tt %]
+
+<h1>[% l('Telegram Bot') %]</h1>
+
+<p>[% l("In order to access <strong>Telegram Bot</strong>, you'll first need to generate an API Token then enter it in the input box below.") %]</p>
```

Comment:
is this the final (approved) wording? to me if we add and `and` to make it `you'll first need to generate an API Token and then enter it` or any such connective, it makes it flow better